src/middlewares/accomodation.middleware.js

```diff
@@ -39,5 +39,31 @@ class AccomodationMiddleware {
     }
     return next();
   }
+
+  /** checks if the accomodation selected is available
+   *  @param {req} req it contains the request of the user
+   *  @param {res} res it contains the response the user receive
+   *  @param {function} next it jumps to the next middleware in the route
+   * @return {object} the data from the first middleware
+   */
+  static async checkBookingRequestAvailability(req, res, next) {
```

Comment:
@VedaDom The name of the function and its description does not match. 
This function is doing more than checking if a given accommodation is available or not.
Kindly update whatever that needs to be updated.